### PR TITLE
Expose application info on ready

### PR DESCRIPTION
**Pull request by bsian03**

### CODE REVIEW

**Comment by macdja38** [comment]
Ik this is new, was it added to gateway v7 as well as v8?

**Comment by bsian03** [comment]
Looks like all versions, I'm running my test bot on v6 gateway and it seems to be populated there

File: lib/Client.js [hunk 1]
```diff
@@ -50,6 +50,7 @@ const sleep = (ms) => new Promise((res) => setTimeout(res, ms));
 /**
 * Represents the main Eris client
 * @extends EventEmitter
+* @prop {Object} application Object containing the bot application's ID and its public flags
```
Comment (alex-taxiera) [inline lib/Client.js]: the types indicate this should be optional

Comment (bsian03) [reply lib/Client.js]:
```suggestion
* @prop {Object} [application] Object containing the bot application's ID and its public flags
```

**Review by Khaaz — DISMISSED** [review]

**Review by abalabahaha — APPROVED** [review]

**Pull request merged**